ci/prow/config.yaml

```diff
@@ -1582,6 +1602,47 @@ periodics:
       resources:
         requests:
           memory: "1Gi"
+- cron: "1 9 * * 2" # Run at 02:01PST every Tuesday (09:01 UTC)
+  name: ci-knative-serving-dot-release
+  agent: kubernetes
+  labels:
+    preset-release-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/prow-tests:latest
+      imagePullPolicy: Always
+      args:
+      - "--scenario=kubernetes_execute_bazel"
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=github.com/knative/serving"
+      - "--root=/go/src"
+      - "--service-account=/etc/release-account/service-account.json"
+      - "--upload=gs://knative-prow/logs"
+      - "--timeout=90" # 1.5h
+      - "--" # end bootstrap args, scenario args below
+      - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+      - "./hack/release.sh"
+      - "--dot-release"
+      - "--release-gcs=knative-releases/serving"
+      - "--release-gcr=gcr.io/knative-releases"
+      - "--github-token=/etc/hub-token/token"
+      # Bazel needs privileged mode in order to sandbox builds.
```

Comment:
We don't use Bazel, so I wonder if we can drop all of these privileged containers from our jobs?

Reply:
We use Bazel to parse results for Testgrid/Gubernator. But I confess that I haven't bothered fine tunning settings again for the past 6 months, probably it's a good time to do it again. Filed #326.